Gems/ROS2/Code/Include/ROS2/Frame/ROS2FrameSystemBus.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/Policies.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/RTTI/RTTIMacros.h>
+#include <AzCore/std/string/string.h>
+#include <ROS2/Frame/ROS2FrameComponent.h>
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+
+namespace ROS2
+{
+    class ROS2FrameSystemRequests
+    {
+    public:
+        AZ_RTTI(ROS2FrameSystemRequests, "{24fe4584-0499-4a37-bc1a-00ca04bd22f5}");
+
+        //! Registers the ROS2FrameEditorComponent into the system. All ROS2FrameEditorComponents should
```

Comment:
Into what system? Since this is a public API, there should be an explanation why this is needed / what is the use.